green_city/tests/api/add_eco_news_test.py

```
@@ -0,0 +1,67 @@
+import pytest
+import json
+import requests
+from jsonschema import validate
+from green_city.config.config import API_BASE_URL_8085, ENDPOINTS
+from green_city.tests.data.schema.add_news_schema import post_schema
+import green_city.config.logging_config
```

Ruff (0.11.9)

`green_city.config.logging_config` imported but unused

Remove unused import: `green_city.config.logging_config`

(F401)